ci-operator/step-registry/baremetalds/assisted/operator/bmh/baremetalds-assisted-operator-bmh-commands.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ baremetalds assisted operator bmh command ************"
+
+# Fetch packet basic configuration
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/packet-conf.sh"
+
+cat "${SHARED_DIR}/ds-vars.conf"
+
+ssh "${SSHOPTS[@]}" "root@${IP}" bash - << "EOF" |& sed -e 's/.*auths\{0,1\}".*/*** PULL_SECRET ***/g'
+
+echo "export SSH_PUBLIC_KEY='${SSH_PUBLIC_KEY:-$(cat $HOME/.ssh/id_rsa.pub)}'" >> /root/config
+echo "export SSH_PRIVATE_KEY='SSH_PRIVATE_KEY:-$(cat $HOME/.ssh/id_rsa)'" >> /root/config
+echo "export PULL_SECRET='\$(cat /root/pull-secret)'" >> /root/config
+
+source /root/config
+
+echo "Assisted Installer pods"
+oc get pods -n assisted-installer
+
+echo "Metal3 pods before SNO install"
+oc get pods -A | grep metal3
+
+echo "Creating new baremetal host using baremetal operator"
+oc create -f ${DS_WORKING_DIR}/extra_host_manifests.yaml
```

Comment:
These files need to be modified to add the labels to match the `infraenv` resource.

I would recommend disabling inspection and cleaning too.